src/mlpack/tests/main_tests/perceptron_test.cpp

```diff
@@ -138,9 +146,7 @@ BOOST_AUTO_TEST_CASE(PerceptronLabelsLessDimensionTest)
   arma::Row<size_t> output;
   output = std::move(CLI::GetParam<arma::Row<size_t>>("output"));
 
-  bindings::tests::CleanMemory();
-
-  // Now train perceptron with labels provided.
+  // Now train pereptron with labels provided.
```

Comment:
I think we should not remove `CleanMemory()` here; is there a reason I have overlooked that we should remove it?

Reply:
Actually, I didn't remove it. The file contained the older tests that Manish had written and the changes that you made to it as per 1214 didn't apply. I will fix that.